examples/reference/panes/Image.ipynb

```diff
@@ -37,7 +37,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The ``Image`` pane can be pointed at any local or remote image file. If given a URL starting with ``http`` or ``https``, the ``embed`` parameter determines whether the image will be embedded or linked to:"
+    "The ``Image`` pane can be pointed at any local or remote image file. If given a URL starting with ``http`` or ``https``, the ``embed`` parameter determines whether the image will be embedded or linked to:
+
+The Image pane is a "macro" pane that selects one of the other more specific image panes to do the actual work (whichever one first recognizes this particular image).
+"
```

Comment:
```suggestion
    "The ``Image`` pane can be pointed at any local or remote image file. If given a URL starting with ``http`` or ``https``, the ``embed`` parameter determines whether the image will be embedded or linked to.

The Image pane is actually a sort of "macro" pane that selects one of the other more specific image panes to do the actual work (whichever one first recognizes this particular image format). See the documentation for the pane for each specific pane (JPG, PNG, SVG, etc.) for the options that particular pane will support.
"
```

Reply:
How about:

The Image pane is actually a sort of "macro" pane that selects one of the other more specific image panes to display the image  (whichever one first recognizes this particular image format).

Please see the documentation for the pane for each specific pane (JPG, PNG, SVG, etc.) for the options that particular pane will support.

Comment: does that mean that using  those specific options will work or not work with Image.Pane, depending on the image format?

Reply:
The added text brings in a bunch of low-level details ("get_pane_type", "classmethod", etc.) that don't seem appropriate for a gallery page.  I think a Pane type's gallery page should focus very specifically on what it does, and how a user should configure that.  So here, mentioning the dispatch mechanism is useful, because that conveys both, but otherwise I don't think it should get into _how_ that works, which is a topic for a reference page rather a user page.

I'm not sitting somewhere where I can test this, but I'd suggest testing Panel yourself to see the answer to the last question -- can you provide JPG-specific options to the Image pane? I don't know the answer to that myself, but in general Panel does dispatch the arguments down to the lower-level object, so I'd expect that to happen here too.

Reply:
Ok I included that because it provides a hint to the logical question someone reading this might have: ok, so how can I see which pane is selected by the Image pane?

So remove this sentence:

"The classmethod get_pane_type returns the pane_type that was selected."

But keep the links to API and source code of the component? It seems logical and natural to include links to these.

Ideally these 2 links should be in every component page. Maybe at the top?